packages/cloud/package.json

```diff
@@ -24,6 +24,7 @@
   },
   "dependencies": {
     "@logto/cli": "workspace:*",
+    "@logto/connector-kit": "workspace:1.0.0-rc.2",
```

Comment:
It looks weird that lock the dependency to 1.0.0-rc.2 and hence refactored the @logto/connector-kit.
Maybe add a comments and update the dependency as soon as the new @logto/connector-kit release is available?

Reply:
updated to *